azure/image/ubuntu/azure-ubuntu.pkr.hcl

```
@@ -30,6 +30,7 @@ source "azure-arm" "ubuntu" {
     image_name           = "${var.az_gallery_image_name}"
     image_version        = "${var.az_gallery_image_version}"
     storage_account_type = "Standard_LRS"
+    replication_regions  = "eastus2"
```

note: replication_regions is an array, I think it makes sense to add eastus2 for TDX availability, we can also add `westeurope`, since it supports both tdx and snp vms
```suggestion
    replication_regions  = ["eastus2", "westeurope"]
```

---

@mkulke added, thanks for the feedback!